Roles/Core/AssignManager/GhostRoleAssign.cs

```diff
@@ -112,5 +112,12 @@ public static void Add()
         if (Options.CustomGhostRoleCounts.Any())
             Options.CustomGhostRoleCounts.Keys.Do(ghostRole
                 => getCount.TryAdd(ghostRole, ghostRole.GetCount())); // Add new count Instance (Optionitem gets constantly refreshed)
+
+
+
+        foreach (var role in getCount)
+        {
+            Logger.Info($"Logged: {role.Key} / {role.Value}", "GhostAssignPatch.Add.GetCount");
+        }
```

Comment:
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/143632280/4f4d77d5-ef5e-4788-ab64-dd73a21ea982)
smt weird happened in someon'es game where apparently bloodmoon was not present in getcount